linebot/event.go

```diff
@@ -179,14 +179,16 @@ type StickerResourceType string
 
 // StickerResourceType constants
 const (
-	StickerResourceTypeStatic         StickerResourceType = "STATIC"
-	StickerResourceTypeAnimation      StickerResourceType = "ANIMATION"
-	StickerResourceTypeSound          StickerResourceType = "SOUND"
-	StickerResourceTypeAnimationSound StickerResourceType = "ANIMATION_SOUND"
-	StickerResourceTypePerStickerText StickerResourceType = "PER_STICKER_TEXT"
-	StickerResourceTypePopup          StickerResourceType = "POPUP"
-	StickerResourceTypePopupSound     StickerResourceType = "POPUP_SOUND"
-	StickerResourceTypeNameText       StickerResourceType = "NAME_TEXT"
+	StickerResourceTypeStatic            StickerResourceType = "STATIC"
+	StickerResourceTypeAnimation         StickerResourceType = "ANIMATION"
+	StickerResourceTypeSound             StickerResourceType = "SOUND"
+	StickerResourceTypeAnimationSound    StickerResourceType = "ANIMATION_SOUND"
+	StickerResourceTypePerStickerText    StickerResourceType = "PER_STICKER_TEXT"
+	StickerResourceTypePopup             StickerResourceType = "POPUP"
+	StickerResourceTypePopupSound        StickerResourceType = "POPUP_SOUND"
+	StickerResourceTypeNameText          StickerResourceType = "NAME_TEXT"
+	StickerResourceTypePerStickerMessage StickerResourceType = "MESSAGE"
```

Comment:
My bad in previous review, it should be `StickerResourceTypeMessage` , and please remove `StickerResourceTypePerStickerText` and `StickerResourceTypeNameText`.